ksqldb-engine/src/test/java/io/confluent/ksql/function/udf/math/ExpTest.java

```diff
@@ -39,22 +40,22 @@ public void shouldHandleNull() {
 
   @Test
   public void shouldHandleNegative() {
-    assertThat(udf.exp(-1), is(0.36787944117144233));
-    assertThat(udf.exp(-1L), is(0.36787944117144233));
-    assertThat(udf.exp(-1.0), is(0.36787944117144233));
+    assertThat(udf.exp(-1), is(closeTo(0.36787944117144233, 0.00000000000000005)));
+    assertThat(udf.exp(-1L), is(closeTo(0.36787944117144233, 0.00000000000000005)));
+    assertThat(udf.exp(-1.0), is(closeTo(0.36787944117144233, 0.00000000000000005)));
   }
 
   @Test
   public void shouldHandleZero() {
-    assertThat(udf.exp(0), is(1.0));
-    assertThat(udf.exp(0L), is(1.0));
-    assertThat(udf.exp(0.0), is(1.0));
+    assertThat(udf.exp(0), is(closeTo(1.0, 0.1)));
+    assertThat(udf.exp(0L), is(closeTo(1.0, 0.1)));
+    assertThat(udf.exp(0.0), is(closeTo(1.0, 0.1)));
   }
 
   @Test
   public void shouldHandlePositive() {
-    assertThat(udf.exp(1), is(2.718281828459045));
-    assertThat(udf.exp(1L), is(2.718281828459045));
-    assertThat(udf.exp(1.0), is(2.718281828459045));
+    assertThat(udf.exp(1), is(closeTo(2.718281828459045, 0.00000000000005)));
```

Comment:
Can you please use scientific notation for errors tolerance? For example:

```java
assertThat(udf.exp(1), is(closeTo(2.718281828459045, 5E-17)))
```

 is more readable than 

```java
assertThat(udf.exp(1), is(closeTo(2.718281828459045, 0.00000000000005)))
```